src/import-data/ImportDataDestination.ts

```diff
@@ -358,15 +363,20 @@ export const ImportDataDestination = (props: ImportDataDestinationProps): ReactN
               h(NewWorkspaceModal, {
                 requiredAuthDomain: requiredAuthorizationDomain,
                 cloudPlatform: getCloudPlatformRequiredForImport(importRequest),
-                renderNotice: ({ selectedBillingProject }) =>
-                  h(Fragment, [
-                    isProtectedData &&
-                      selectedBillingProject?.cloudPlatform === 'AZURE' &&
-                      p([
+                renderNotice: ({ selectedBillingProject }) => {
+                  const children: ReactElement[] = [];
```

Comment:
Nit: suggest the broader `ReactNode` type instead of `ReactElement`.

```suggestion
                  const children: ReactNode[] = [];
```

Reply:
Done